canvas/templates/canvas/event_create.html

```diff
@@ -13,30 +13,36 @@
 {% endblock %}
 
 {% block header %}
-    Create Event for {{ course.name }}
+    {% if header %}
+        {{ header }} Event for {{ event.course.name }}
+    {% else %}
+        Create Event for {{ course.name }}
```

Comment:
shouldn't this be {{event.course.name}} ?

Reply:
When the template is used to create an event, the event object isn't passed in from the create_event_view (because the event doesn't exist yet), but it does pass in the course object